### PR TITLE
Week 2 answers from Robert

### PR DESCRIPTION
**Pull request by psytau**
The homework is here. I had a question about private and modules. How would that work?

### CODE REVIEW

File: week2/homework/simon_says.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  def first_word(str)
+    tokens = tokenize(str)
+    tokens[0]
+  end
+
+  def tokenize(str)
+    str.split(/\W/)
+  end
+
```
Comment (psytau) [inline week2/homework/simon_says.rb:34]: Can I make tokenize private? Would that be a good idea?

Comment (bethanyr) [inline week2/homework/simon_says.rb:34]: yes - if you are only calling tokenize from within your module, then you can make it private - and that would be a good practice.

File: week2/homework/simon_says.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  def echo(str)
+    str
+  end
+
+  def shout(str)
+    str.upcase
+  end
+
+  def repeat(str, repeat_times=2)
```
Comment (bethanyr) [inline week2/homework/simon_says.rb:13]: this is one way to code it, Ruby provides a simpler syntax to do the same thing - take a look at the answers for a simpler way.

**Comment by bethanyr** [comment]
I reviewed week 2 homework - looks good.

File: week2/homework/simon_says.rb (hunk 2 above)
Comment (psytau) [inline week2/homework/simon_says.rb:13]: Thanks for the comment! I see... [str]*n.

**Pull request closed without merging**